src/react/outline.md

```diff
@@ -0,0 +1,78 @@
+@page react/outline React Content Outline
+@parent react 1
+
+@body
+
+## Intro
+
+* Before you begin
+  * Who is this guide for?
+    * Know the basics of Javascript, including ES2016 (include brief review)
```

Comment:
Unless you have an idea about the `.includes()` method or the exponential operator (`**`) I suspect you either mean **ES2015** or maybe **ES2018** (property spread is pretty widely used), or maybe just **ES.next** maybe?